docs/ReleaseProcess.md

```diff
@@ -76,10 +106,11 @@ Servicing a release has two main scenarios:
     >```bash
     >git commit --no-verify
     >```
-1. Create a **Draft** pull-request for your working branch into the main repo's "`release/X.Y`" branch, and title the PR as *"Release v`X.Y.Z`"* (replace "`X.Y.Z`" with the version number being released).
+1. Create a **Draft** pull-request from the topic branch into the upstream repo's "`release/X.Y`" branch, and title the PR as *"Release v`X.Y.Z`"* where "`Z`" is the patch version.
 1. Wait for  the CI/CD pipeline for the PR to complete successfully.
-1. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`release/X.Y`" branch.
-1. Publish the release as per the "[Publishing a Release](ReleaseProcess.md#publishing-a-release)" process.
+1. Mark the draft PR as "Ready for review".
```

Comment:
I'm not sure why you have the process of using Draft PRs and then publishing them. You should eliminate as many manual steps as possible. The automation itself should enforce when things are ready. Draft PRs should only be used when you are sure there will be additional commits, and it is not worth review from someone.

Reply:
Agreed. simplified.